tests/safety/test_honda.py

```diff
@@ -428,10 +432,11 @@ def _acc_state_msg(self, main_on):
     self.__class__.cnt_acc_state += 1
     return self.packer.make_can_msg_panda("SCM_BUTTONS", self.PT_BUS, values)
 
-  def _button_msg(self, buttons, main_on=False):
+  def _button_msg(self, buttons, main_on=False, bus=None):
+    bus = self.PT_BUS if bus is None else bus
```

Comment:
why not just the default to self.PT_BUS? also is this even used anywhere?

Reply:
oh looks like you just want to use self.BUTTONS_BUS?

Reply:
No, we need to receive buttons on bus 0 and send on bus 2 so can't just use one bus.

Reply:
Also can't default to self variables